doc/pages/administrator/030-installation_kubernetes.md

```diff
@@ -1,41 +1,93 @@
 ```
 
-This command will take care of installing all needed components for the Operator to run. This includes all the
-RBAC roles, Custom Resource Definitions, and the Operator itself. The `--version` switch allows to specify a
-version to install - when not specified, the latest version will be installed instead.
+This command will take care of installing all needed components for the Operator to run. This
+includes all the RBAC roles, Custom Resource Definitions, and the Operator itself. The `--version`
+switch allows to specify a version to install - when not specified, the latest version will be
+installed instead.
 
 ## Upgrading the Operator
 
-To upgrade the Operator, use the dedicated `upgrade-operator` command.
+The procedure for upgrading the Operator depends on the version of the Operator you want to
+upgrade from.
+
+### Upgrading the Operator (>= v1.0)
+
+Astarte Operator's Helm chart is available since v1.0 and therefore the current upgrade procedure
+applies only if you are dealing with at least v1.0.
+
+To upgrade the Operator, use the dedicated `helm upgrade` command:
 
 ```bash
-astartectl cluster upgrade-operator
+helm upgrade astarte-operator astarte/astarte-operator [--version <new-version>]
 ```
 
-Just like the `install-operator` command, the `--version` switch allows to specify the version to upgrade to -
-when not specified, the latest version will be fetched and used.
+The optional `--version` switch allows to specify the version to upgrade to - when not specified,
+the latest version will be fetched and used.
+
+Currently, only the first version of the chart is available within `helm.astarte-platform.org` and
+the `helm upgrade` operation will lead to an error. Bear with that until the next version gets
+released.
+
+### Upgrading the Operator (from v0.11 to v1.0)
+
+The upgrade procedure from v0.11 to v1.0 requires some manual intervention as the deployment and
+handling of the Operator's lifecylcle has changed: if v0.11 is entirely handled with `astartectl`,
+v1.0 employs Helm charts.
+
+To ensure a consistent upgrade procedure, the following steps are required:
+
++ Remove the Operator's Service Account, Cluster Roles and Cluster Role Bindings:
+```bash
+kubectl delete serviceaccounts -n kube-system astarte-operator
+kubectl delete clusterroles.rbac.authorization.k8s.io astarte-operator
+kubectl delete clusterrolebindings.rbac.authorization.k8s.io astarte-operator
+```
+
++ Delete the Operator's deployment:
+```bash
+kubectl delete deployments.app -n kube-system astarte-operator
+```
+
+**DO NOT** delete Astarte's CRDs! This will lead to the deletion of the entire astarte deployment
+with a consequent data loss.
+
+To restore the Operator's functionalities with v1.0, all you have to do is following the install
+instructions, i.e.:
+```bash
+helm repo add astarte https://helm.astarte-platform.org
+helm repo update
+helm install astarte-operator astarte/astarte-operator --version 1.0.0-alpha.1
```

Comment:
The version here probably makes sense instead, as we want to provide a safe landing when crossgrading. Ideally we should mention that (aka: all upgrades from 0.x have to go to $version, then they can be upgraded to latest)